src/app/clusters/level-control/level-control.cpp

```diff
@@ -267,8 +267,8 @@ void emberAfLevelControlClusterServerTickCallback(EndpointId endpoint)
         currentLevel.SetNonNull(static_cast<uint8_t>(currentLevel.Value() - 1));
     }
 
-    ChipLogProgress(Zcl, " to %d ", currentLevel.Value());
-    ChipLogProgress(Zcl, "(diff %c1)", state->increasing ? '+' : '-');
+    ChipLogDetail(Zcl, " to %d ", currentLevel.Value());
```

Comment:
I wonder why this message is split into 3 lines instead of printing everything in a single line...

Reply:
Somebody wanted it to stand out...